.mailmap

```diff
@@ -89,6 +89,7 @@ Jean-Baptiste Dalido <jeanbaptiste@appgratis.com>
 Sven Dowideit <SvenDowideit@home.org.au>
 Sven Dowideit <SvenDowideit@home.org.au> <SvenDowideit@fosiki.com>
 Sven Dowideit <SvenDowideit@home.org.au> <SvenDowideit@docker.com>
+Sven Dowideit <SvenDowideit@home.org.au> <¨SvenDowideit@home.org.au¨>
 Sven Dowideit <SvenDowideit@home.org.au> <SvenDowideit@home.org.au>
```

Comment:
@SvenDowideit Yes somewhere there's a commit with this, so adding it here prevents it from ending up in the Authors file :+1: